### DISCUSSION TITLE
How Will Loss of Separation Alarm be triggered?

### DISCUSSION THREAD

**Discussion by crixus5678** [TUDelft-CNS-ATM/bluesky#532] [General]
Hi,

I tried to run a simulation whereby I place two airplanes in the same exact location just to check if the LoS (loss of separation alarm will be triggered) but on the GUI, it doesn't seem like there is any alarm (for example LoS is still showing as 0/0 - i assume los here refers to loss of separation?)

0:00:00.00>Cre KL204, A320, 2.7456, 101.7092, 140, 2000, 150 
0:00:00.00>Cre KL205, A320, 2.7456, 101.7092, 140, 2000, 150

**Comment by jooste** [comment]
Hi @crixus5678,

You have to first turn on conflict detection: `ASAS ON`. After that, state-based conflicts and losses of separation will be detected once a second.

**Reply by crixus5678** [reply]
thanks a lot of the fast reply. I assume all the available commands, condition of loss of separation and description of any other function will be located in the following link?

https://github.com/TUDelft-CNS-ATM/bluesky/tree/master/docs

**Reply by jooste** [reply]
Not everything is documented, but for most commands you can find documentation [here](https://github.com/TUDelft-CNS-ATM/bluesky/wiki/Command-Reference). The same documentation can be found inside of bluesky, by using the `DOC` command.

**Comment by zimbofly** [comment]
Hi,
is there also a way to monitor separation losses/violations within .log files (using periodic loggers)?

Kind regards
